dataland-frontend/src/components/forms/parts/elements/basic/MultiSelectFormElement.vue

```diff
@@ -47,6 +50,14 @@ export default defineComponent({
       selections: [] as string[],
     };
   },
+  watch: {
+    selections(newVal) {
+      this.$emit("selectedValuesChanged", newVal);
+    },
+    modelValue(newVal: []) {
+      this.selections = newVal;
+    },
+  },
   props: {
```

Comment:
They way the props are defined here is not consistent with our current strategy.
The correct way would be to directly import the `FormFieldProps` from the file `FormFieldProps.ts`.
If those pre-defined `FormFieldProps` don't have enough properties to fulfill the needs in this component, we will have to write an extension for the `MultiSelectFormElement` component inside of the `FormFieldProps.ts`

Reply:
We've got the same issue also for the `SingleSelectFormElement.vue`.   There we also directly code props, instead of importing it from `FormFieldProps.ts`.

Reply:
Done.

Reply:
👍